apps/ledger-live-mobile/src/families/ethereum/EditTransaction/EditOperationPanel.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { memo, useCallback } from "react";
+import { useTranslation } from "react-i18next";
+import { Box, Flex } from "@ledgerhq/native-ui";
+import { useFeature } from "@ledgerhq/live-common/featureFlags/index";
+import { useNavigation } from "@react-navigation/core";
+import { Account, AccountLike, Operation } from "@ledgerhq/types-live";
+import { getStuckAccountAndOperation } from "@ledgerhq/coin-framework/operation";
+
+import LText from "../../../components/LText";
+import Link from "../../../components/wrappedUi/Link";
+import { NavigatorName, ScreenName } from "../../../const";
+
+type EditOperationPanelProps = {
+  isOperationStuck: boolean;
+  operation: Operation;
+  account: AccountLike | null | undefined;
+  parentAccount: Account | null | undefined;
+  onPress?: () => void;
+};
+
+const EditOperationPanelComponent = ({
+  isOperationStuck,
+  operation,
+  account,
+  parentAccount,
+  onPress,
+}: EditOperationPanelProps) => {
+  const { t } = useTranslation();
+  const flag = useFeature("editEthTx");
+  const navigation = useNavigation();
+
+  const onLinkPress = useCallback(() => {
+    onPress && onPress();
+
+    if (account) {
+      navigation.navigate(NavigatorName.EditTransaction, {
+        screen: ScreenName.EditTransactionMethodSelection,
+        params: { operation, account, parentAccount },
+      });
+    }
+  }, []);
+
+  const stuckAccountAndOperation = account
+    ? getStuckAccountAndOperation(account, parentAccount)
+    : undefined;
```

Comment:
I'm not sure to understand here. We're using the `getStuckAccountAndOperation` but from what I can see in the props, you already have the `isOperationStuck` + `operation, account, parentAccount`, so why are we using it again here ?
Also if we need to use it, shouldn't the return be used as parameters to the `onLinkPress` ?